package.json

```diff
@@ -213,7 +213,7 @@
                 },
                 {
                     "command": "aks.installAzureServiceOperator",
-                    "when": "view == kubernetes.cloudExplorer && viewItem =~ /aks\\.cluster/i"
+                    "when": "view == kubernetes.cloudExplorer && viewItem =~ /aks\\.cluster/i || view == extension.vsKubernetesExplorer && viewItem =~ /vsKubernetes\\.\\w*cluster$/i"
```

Comment:
💡❤️☕️🎅 For arichve reasons, and please correct me if it's not entirely correct, this work, we will document this as a final release PR for how this enables the ASO for kind cluster et. al. as well and hence a hook to `k8s cncf` extension as well. 

I think this doc doesn't need to be in other extension because its hook which user will only get is the `k8s extension` is downloaded.

Reply:
I think that's correct. It enables ASO for non-AKS clusters (whatever appears in cluster explorer).

I don't think we need any docs in the k8s extension. This makes use of [an extension point](https://github.com/vscode-kubernetes-tools/vscode-kubernetes-tools/blob/master/docs/extending/commandtargets.md#cluster-explorer-node-right-click) for adding commands to the tree view, but the command itself is here.